docs/installation/est/Set-Up-Realm-DB.adoc

```diff
@@ -30,7 +30,7 @@ require to modify the authorization script
 
 
 If you have chosen to use *PostgreSQL* for user management, you first
-need to prepare a database (e.g. est) to access the
+need to prepare a database, for example est, to access the
```

Comment:
Since we're not using parentheses anymore, we probably should enclose est in backticks (i.e. `est`) to clarify that it's an identifier, not an English word, to avoid confusions.